### PR TITLE
Include camel-k-runtime version in camel-catalog

### PR DESCRIPTION
**Pull request by lburgazzoli**
Fixes #959

### CODE REVIEW

File: tooling/camel-k-maven-plugin/src/it/generate-catalog/verify.groovy [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
```
Comment (WillemJiang) [inline tooling/camel-k-maven-plugin/src/it/generate-catalog/verify.groovy]: The license header need to be updated.

Comment (lburgazzoli) [reply tooling/camel-k-maven-plugin/src/it/generate-catalog/verify.groovy]: done

**Comment by oscerd** [comment]
Sorry, I merged but I was looking at a different tab.. If needed I can revert

**Comment by lburgazzoli** [comment]
no worries, I was faster than you :)

**Pull request merged**